cmd/tailscaled/debug.go

```diff
@@ -131,7 +131,10 @@ func getURL(ctx context.Context, urlStr string) error {
 }
 
 func checkDerp(ctx context.Context, derpRegion string) error {
-	dmap := derpmap.Prod()
+	dmap, err := tailscale.ProdDERPMap(ctx, &http.Client{})
```

Comment:
I'd just inline the fetch-from-network here. Use `ipn.DefaultControlURL` instead of hard-coding the hostname again.

Reply:
er, do you mean inline all the code from ProdDERPMap? Is there any particular reason to do that?

Reply:
Let me know if this is sufficient, there are two near identical copies of the same query in the codebase, but if that's fine then should be good